com.unity.formats.fbx/Editor/FbxExporter.cs

```diff
@@ -1199,42 +1197,68 @@ private bool ExportSkeleton (SkinnedMeshRenderer skinnedMesh, FbxScene fbxScene,
         /// <summary>
         /// set vertex weights in cluster
         /// </summary>
-        private void SetVertexWeights (MeshInfo meshInfo, Dictionary<int, FbxCluster> boneCluster)
+        private void SetVertexWeights (MeshInfo meshInfo, Dictionary<int, FbxCluster> boneIndexToCluster)
         {
-            HashSet<int> visitedVertices = new HashSet<int> ();
+            var mesh = meshInfo.mesh;
```

Comment:
changes are mostly taken from the documentation: https://docs.unity3d.com/ScriptReference/Mesh.GetAllBoneWeights.html